cmd/juno/dbcmd.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/NethermindEth/juno/blockchain"
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/db"
+	"github.com/NethermindEth/juno/db/pebble"
+	"github.com/NethermindEth/juno/utils"
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+)
+
+type DBInfo struct {
+	Network         string     `json:"network"`
+	ChainHeight     uint64     `json:"chain_height"`
+	LatestBlockHash *felt.Felt `json:"latest_block_hash"`
+	LatestStateRoot *felt.Felt `json:"latest_state_root"`
+	L1Height        uint64     `json:"l1_height"`
+	L1BlockHash     *felt.Felt `json:"l1_block_hash"`
+	L1StateRoot     *felt.Felt `json:"l1_state_root"`
+}
+
+func DBCmd() *cobra.Command {
+	dbCmd := &cobra.Command{
+		Use:   "db",
+		Short: "Database related operations",
+		Long:  `This command allows you to perform database operations.`,
+	}
+	dbCmd.PersistentFlags().StringP(dbPathF, "p", "", dbPathUsage)
+	dbCmd.AddCommand(DBInfoCmd(), DBSizeCmd())
+	return dbCmd
+}
+
+func DBInfoCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "info",
+		Short: "Retrieve database information",
+		Long:  `This subcommand retrieves and displays blockchain information stored in the database.`,
+		RunE:  dbInfo,
+	}
+}
+
+func DBSizeCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "size",
+		Short: "Calculate database size information for each data type",
+		Long:  `This subcommand retrieves and displays the storage of each data type stored in the database.`,
+		RunE:  dbSize,
+	}
+}
+
+func dbInfo(cmd *cobra.Command, args []string) error {
+	dbPath, err := cmd.Flags().GetString(dbPathF)
```

Comment:
What do you think about checking the default path $HOME/juno if no flag is set?

Reply:
do we store it at this location if user doesn't specify `--db-path` ?

Reply:
The default db path (for db-related subcommands) is `""`.

I added a change to pass the default db path from `junoCmd` to `dbCmd`.